.test-infra/jenkins/README.md

```diff
@@ -88,28 +88,30 @@ Beam Jenkins overview page: [link](https://builds.apache.org/view/A-D/view/Beam/
-| beam_Inventory_beam7 | [cron](https://builds.apache.org/job/beam_Inventory_beam7/) | `Run inventory on beam7` | [![Build Status](https://builds.apache.org/job/beam_Inventory_beam7/badge/icon)](https://builds.apache.org/job/beam_Inventory_beam7) |
-| beam_Inventory_beam8 | [cron](https://builds.apache.org/job/beam_Inventory_beam8/) | `Run inventory on beam8` | [![Build Status](https://builds.apache.org/job/beam_Inventory_beam8/badge/icon)](https://builds.apache.org/job/beam_Inventory_beam8) |
-| beam_Inventory_beam9 | [cron](https://builds.apache.org/job/beam_Inventory_beam9/) | `Run inventory on beam9` | [![Build Status](https://builds.apache.org/job/beam_Inventory_beam9/badge/icon)](https://builds.apache.org/job/beam_Inventory_beam9) |
-| beam_Inventory_beam10 | [cron](https://builds.apache.org/job/beam_Inventory_beam10/) | `Run inventory on beam10` | [![Build Status](https://builds.apache.org/job/beam_Inventory_beam10/badge/icon)](https://builds.apache.org/job/beam_Inventory_beam10) |
-| beam_Inventory_beam11 | [cron](https://builds.apache.org/job/beam_Inventory_beam11/) | `Run inventory on beam11` | [![Build Status](https://builds.apache.org/job/beam_Inventory_beam11/badge/icon)](https://builds.apache.org/job/beam_Inventory_beam11) |
-| beam_Inventory_beam12 | [cron](https://builds.apache.org/job/beam_Inventory_beam12/) | `Run inventory on beam12` | [![Build Status](https://builds.apache.org/job/beam_Inventory_beam12/badge/icon)](https://builds.apache.org/job/beam_Inventory_beam12) |
-| beam_Inventory_beam13 | [cron](https://builds.apache.org/job/beam_Inventory_beam13/) | `Run inventory on beam13` | [![Build Status](https://builds.apache.org/job/beam_Inventory_beam13/badge/icon)](https://builds.apache.org/job/beam_Inventory_beam13) |
-| beam_Inventory_beam14 | [cron](https://builds.apache.org/job/beam_Inventory_beam14/) | `Run inventory on beam14` | [![Build Status](https://builds.apache.org/job/beam_Inventory_beam14/badge/icon)](https://builds.apache.org/job/beam_Inventory_beam14) |
-| beam_Inventory_beam15 | [cron](https://builds.apache.org/job/beam_Inventory_beam15/) | `Run inventory on beam15` | [![Build Status](https://builds.apache.org/job/beam_Inventory_beam15/badge/icon)](https://builds.apache.org/job/beam_Inventory_beam15) |
-| beam_Inventory_beam16 | [cron](https://builds.apache.org/job/beam_Inventory_beam16/) | `Run inventory on beam16` | [![Build Status](https://builds.apache.org/job/beam_Inventory_beam16/badge/icon)](https://builds.apache.org/job/beam_Inventory_beam16) |
+| beam_Inventory_apache-beam-jenkins-1 | [cron](https://builds.apache.org/job/beam_Inventory_apache-beam-jenkins-1/) | `Run inventory apache-beam-jenkins-1` | [![Build Status](https://builds.apache.org/job/beam_Inventory_apache-beam-jenkins-1/badge/icon)](https://builds.apache.org/job/beam_Inventory_apache-beam-jenkins-1) |
```

Comment:
I forgot changing these after switching to the new workers. Thanks for making them correct.

Reply:
It's hard for people to update this README if they touches groovy files. Probably we should automate this file or setup an alert. On the other hand, [Grafana dashboard](http://104.154.241.245/?orgId=1) can do the same thing.